dll/win32/shell32/folders/CRegFolder.cpp

```diff
@@ -82,6 +82,17 @@ HRESULT CALLBACK RegFolderContextMenuCallback(IShellFolder *psf,
         if (!SH_ShowRecycleBinProperties(L'C'))
             hr = E_FAIL;
     }
+    else if (_ILIsMyDocuments(apidl[0]))
+    {
+        WCHAR szPath[MAX_PATH];
+
+        /* FIXME: Windows has the special "Target" page for My Documents properties dialog */
+        hr = SHGetFolderPathW(NULL, CSIDL_MYDOCUMENTS, NULL, SHGFP_TYPE_CURRENT, szPath);
+        if (SUCCEEDED(hr))
+        {
+            hr = (SH_ShowPropertiesDialog(szPath, pdtobj) ? S_OK : E_FAIL);
+        }
+    }
```

Comment:
I'd have rather that you turn this into and `else` case that sets `hr = S_FALSE`,
which will tell the calling code to run the default action.

Reply:
The result will be:
![Capture](https://user-images.githubusercontent.com/86486978/204423544-c196b22d-551b-4f78-8c9f-46835867b053.PNG)

I've added else case to tell the caller to run the default action.

Reply:
But you still left the specialization for the 'My Documents' case.

Reply:
At least `else if (_ILIsMyDocuments(apidl[0]))` is correct because My Documents has a special dialog.
The dialog can be implemented later.

Reply:
No, special dialogs don't have to be hardcoded like that, they are most likely from a shell extension. (With some exceptions)

For example, the 'compatibility' dialog is not hardcoded either.

Reply:
OK.